flutter_app/pubspec.lock

```diff
@@ -137,6 +163,34 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "2.0.5"
+  shared_preferences:
+    dependency: "direct main"
+    description:
+      name: shared_preferences
+      url: "https://pub.dartlang.org"
+    source: hosted
+    version: "0.5.6+3"
+  shared_preferences_macos:
```

Comment:
this seems like a non-flutter package. is this even needed when running in the simulator?